geniza/corpus/models.py

```diff
@@ -527,6 +527,14 @@ def shelfmark_display(self):
             return None
         return certain[0] + (" + …" if len(certain) > 1 else "")
 
+    @property
+    def historical_date_display(self):
```

Comment:
please add a docstring

Reply:
can we just rename to `original_date` or similar? display seems unnecessary since it doesn't conflict with any fields